src/default_out.rs

```diff
@@ -1141,15 +1145,15 @@ pub fn dump_undefined_struct(
                 );
             }
             for installable_language in data.installable_langauges() {
```

Comment:
Shouldn't SMBiosStringSet should iterate over elements of SMBiosString? Instead of `Result<String, FromUtf8Error>`

Reply:
OK, have a look now.  This should build without errors.  With the new SMBiosString being returned, I replaced these spots with a .to_utf8_lossy().unwrap_or_default().  I also added code for handling the new CacheSize...

Reply:
Oh, the I already implemented cache size fixes in https://github.com/jrgerber/dmidecode-rs/pull/54. But I think this iterator change is the right one.